commit 1bf49f3a2682696fa4d4834ae026feb50de21053
Author: Surya Chittuluri <suryanar@codeaurora.org>
Date:   Tue Jan 31 15:58:15 2012 -0800

    decreased the static buffers used for logging (from 1MB to 256KB)
    
    Change-Id: I02ab0e83b2f65bf7bbafe4ac20fdef3456103781
    Signed-off-by: Surya Chittuluri <suryanar@codeaurora.org>

diff --git a/drivers/staging/android/logger.c b/drivers/staging/android/logger.c
index fa76ce7..88e3547 100644
--- a/drivers/staging/android/logger.c
+++ b/drivers/staging/android/logger.c
@@ -555,10 +555,10 @@ static struct logger_log VAR = { \
 	.size = SIZE, \
 };
 
-DEFINE_LOGGER_DEVICE(log_main, LOGGER_LOG_MAIN, 256*1024)
-DEFINE_LOGGER_DEVICE(log_events, LOGGER_LOG_EVENTS, 256*1024)
-DEFINE_LOGGER_DEVICE(log_radio, LOGGER_LOG_RADIO, 256*1024)
-DEFINE_LOGGER_DEVICE(log_system, LOGGER_LOG_SYSTEM, 256*1024)
+DEFINE_LOGGER_DEVICE(log_main, LOGGER_LOG_MAIN, 64*1024)
+DEFINE_LOGGER_DEVICE(log_events, LOGGER_LOG_EVENTS, 64*1024)
+DEFINE_LOGGER_DEVICE(log_radio, LOGGER_LOG_RADIO, 64*1024)
+DEFINE_LOGGER_DEVICE(log_system, LOGGER_LOG_SYSTEM, 64*1024)
 
 static struct logger_log *get_log_from_minor(int minor)
 {
